Monado integration build error

So, following https://illixr.github.io/ILLIXR/getting_started for V2 which looks like V-latest is 2.1.10

`git clone --recursive --branch v2-latest https://github.com/ILLIXR/ILLIXR`

`git submodule update --init --recursive`

`./install_deps.sh -j 8` and answering yes for all except QEMU

``./runner.sh configs/native.yaml` seems to run ok, I get the debug, extended window for HMD, and terminal printout

Issue is trying to use the monado integration 

`./runner.sh configs/monado.yaml`
 
I end up with 

> stderr:
>   /home/lincoln/dev/ILLIXR/.cache/paths/https%c%s%sgithub.com%sILLIXR%smonado_integration.git/src/xrt/drivers/illixr/illixr_device.cpp: In function ‘int illixr_rt_launch(illixr_hmd*, const char*, const char*, void*)’:
>   /home/lincoln/dev/ILLIXR/.cache/paths/https%c%s%sgithub.com%sILLIXR%smonado_integration.git/src/xrt/drivers/illixr/illixr_device.cpp:175:76: error: call of overloaded ‘create(const char*&)’ is ambiguous
>     175 |  dh->runtime_lib = new ILLIXR::dynamic_lib{ILLIXR::dynamic_lib::create(path)};
>         |                                                                            ^
>   In file included from /home/lincoln/dev/ILLIXR/.cache/paths/https%c%s%sgithub.com%sILLIXR%smonado_integration.git/src/xrt/drivers/illixr/illixr_device.cpp:31:
>   /home/lincoln/dev/ILLIXR/runtime/common/dynamic_lib.hpp:39:21: note: candidate: ‘static ILLIXR::dynamic_lib ILLIXR::dynamic_lib::create(const string&)’
>      39 |  static dynamic_lib create(const std::string& path) {
>         |                     ^~~~~~
>   /home/lincoln/dev/ILLIXR/runtime/common/dynamic_lib.hpp:43:21: note: candidate: ‘static ILLIXR::dynamic_lib ILLIXR::dynamic_lib::create(const string_view&)’
>      43 |  static dynamic_lib create(const std::string_view& path) {
>         |                     ^~~~~~
>   /home/lincoln/dev/ILLIXR/.cache/paths/https%c%s%sgithub.com%sILLIXR%smonado_integration.git/src/xrt/drivers/illixr/illixr_device.cpp:175:77: error: no matching function for call to ‘ILLIXR::dynamic_lib::dynamic_lib(<brace-enclosed initializer list>)’
>     175 |  dh->runtime_lib = new ILLIXR::dynamic_lib{ILLIXR::dynamic_lib::create(path)};
>         |                                                                             ^
>   In file included from /home/lincoln/dev/ILLIXR/.cache/paths/https%c%s%sgithub.com%sILLIXR%smonado_integration.git/src/xrt/drivers/illixr/illixr_device.cpp:31:
>   /home/lincoln/dev/ILLIXR/runtime/common/dynamic_lib.hpp:28:2: note: candidate: ‘ILLIXR::dynamic_lib::dynamic_lib(ILLIXR::dynamic_lib&&)’
>      28 |  dynamic_lib(dynamic_lib&& other)
>         |  ^~~~~~~~~~~
>   /home/lincoln/dev/ILLIXR/runtime/common/dynamic_lib.hpp:28:2: note:   conversion of argument 1 would be ill-formed:
>   /home/lincoln/dev/ILLIXR/runtime/common/dynamic_lib.hpp:23:2: note: candidate: ‘ILLIXR::dynamic_lib::dynamic_lib(ILLIXR::void_ptr&&)’
>      23 |  dynamic_lib(void_ptr&& handle)
>         |  ^~~~~~~~~~~
>   /home/lincoln/dev/ILLIXR/runtime/common/dynamic_lib.hpp:23:2: note:   conversion of argument 1 would be ill-formed:
>   make[2]: *** [src/xrt/drivers/CMakeFiles/drv_illixr.dir/build.make:76: src/xrt/drivers/CMakeFiles/drv_illixr.dir/illixr/illixr_device.cpp.o] Error 1
>   make[1]: *** [CMakeFiles/Makefile2:597: src/xrt/drivers/CMakeFiles/drv_illixr.dir/all] Error 2
>   make: *** [Makefile:136: all] Error 2

--- a/CONTRIBUTORS
+++ b/CONTRIBUTORS
@@ -26,5 +26,6 @@ Fang Lu <fanglu2@illinois.edu>
 Joseph Ravichandran <jpr3@illinois.edu>
 Giordano Salvador <e3m3@users.noreply.github.com>
 Finn Sinclair <fns2@illinois.edu>
+Lincoln Stein <lincolntstein@gmail.com>
 Hengzhi Yuan <hengzhi2@illinois.edu>
 Jeffrey Zhang <jfzhang2@illinois.edu>

--- a/configs/monado.yaml
+++ b/configs/monado.yaml
@@ -24,4 +24,7 @@ action:
       git_repo: https://github.com/ILLIXR/Monado_OpenXR_Simple_Example.git
       version: master
     config: {}
+  kimera_path: .cache/paths/https%c%s%sgithub.com%sILLIXR%sKimera-VIO/
+  
+
 profile: opt

--- a/runner/runner/main.py
+++ b/runner/runner/main.py
@@ -217,7 +217,8 @@ def load_monado(config: Mapping[str, Any]) -> None:
             ILLIXR_DEMO_DATA=str(demo_data_path),
             ILLIXR_OFFLOAD_ENABLE=str(enable_offload_flag),
             ILLIXR_ALIGNMENT_ENABLE=str(enable_alignment_flag),
-            ILLIXR_ENABLE_VERBOSE_ERRORS=str(config["enable_verbose_errors"])
+            ILLIXR_ENABLE_VERBOSE_ERRORS=str(config["enable_verbose_errors"]),
+            KIMERA_ROOT=config["action"]["kimera_path"],
         ),
         check=True,
     )

